### DISCUSSION TITLE
Python

### DISCUSSION THREAD

**Discussion by KTechDev4213** [gopro/labs#380] [Feature Requests]
Considering that gopros run on linux would it be possible to have a python interpreter on the gopro?

**Comment by dnewman-gpsw** [comment]
There is only part of the camera to using Linux for the networking, everything is an lighter/faster RTOS. As a Labs developer, I never use the Linux components, all the cooler stuff in on the RTOS.

Maybe explain what you want to do, rather than the preferred tool.

**Comment by KTechDev4213** [comment]
I want to be able to start video when a face is recognized.
Sorry about the late reply.

**Reply by dnewman-gpsw** [reply]
The camera's face detection, is only that, it doesn't do recognition.   It simply counts the number of faces in a frame,  that might technically possible to make a trigger base on face count, but I expect that is on less value.

**Comment by Nickos72** [comment]
guys I have a problem. Can you tell me what is wrong with my code?


import tkinter as tk
import random

def open_new_window():
    new_window = tk.Toplevel(root)  # Create a new Toplevel window
    new_window.title("New Window")
    # Add widgets and configure the new window as needed

root = tk.Tk()  # Create the main window
root.geometry("500x300")
root.title("Guessing Game")

count = 0

def check_guess():
    global count
    guess = int(entry.get())
    number = random.randint(1, 100)

    if guess == number and count != 10:
        tk.Label(root, text="You WON!", font="ar 15").grid(row=3, column=3)   
    elif guess < number and count != 10:
        count += 1
        tk.Label(root, text="Higher", font="ar 15").grid(row=3, column=3)
    elif guess > number and count != 10:
        count += 1
        tk.Label(root, text="Lower", font="ar 15").grid(row=3, column=3)
    elif count == 10:
        tk.Label(root, text="You Lost", font="ar 15").grid(row=3, column=3)

tk.Label(root, text="Say a number from 1 to 100", font="ar 12").grid(row=0, column=3)
tk.Label(root, text="", font="ar 10").grid(row=1, column=3)
tk.Label(root, text="Guess here", font="ar 10").grid(row=2, column=1)

entry = tk.Entry(root)
entry.grid(row=2, column=3)

# Submit button
tk.Button(text="Check guess", command=check_guess).grid(row=2, column=4)

root.mainloop()  # Start the main event loop

**Reply by dnewman-gpsw** [reply]
Not a Labs question

**Reply by Nickos72** [reply]
Thanks
